README.md

```diff
@@ -43,69 +43,95 @@ remotes::install_github("mrc-ide/frogger", upgrade = FALSE)
-| totpop1          | total_population   | Projected total population                               |
-| hivpop1          | hiv_population     | Projected HIV population                                 |
-| infections       | infections         |                                                          |
-| hivstrat_adult   | hiv_strat_adult    |                                                          |
-| artstrat_adult   | art_strat_adult    |                                                          |
-| births           | births             | Projected number of births                               |
-| natdeaths        | natural_deaths     | Projected number of natural deaths                       |
-| natdeaths_hivpop | hiv_natural_deaths | Projected number of natural deaths within HIV population |
-| hivdeaths        | hiv_deaths         |                                                          |
-| aidsdeaths_noart | aids_deaths_no_art |                                                          |
-| aidsdeaths_art   | aids_deaths_art    |                                                          |
-| artinit          | art_initiation     |                                                          |
+Discussion: naming conventions
+
+* Distinguish events vs. counts
+* Distinguish stratification of array (single-year population pXX vs. HIV population hXX
+
+| Leapfrog         | Frogger              | Details                                                                  |
+|------------------|----------------------|--------------------------------------------------------------------------|
+| totpop1          | p_total_population   | Projected total population                                               |
+| hivpop1          | p_hiv_population     | Projected HIV population                                                 |
+| infections       | p_infections         | Projected number of new HIV infections by sex and age                    |
+| hivstrat_adult   | h_hiv_strat_adult    | Projected PLHIV not on ART by age, sex and CD4                           |
+| artstrat_adult   | h_art_strat_adult    | Projected PLHIV on ART by age, sex, CD4 and treatment stage              |
+| births           | births               | Projected number of births                                               |
+| natdeaths        | p_natural_deaths     | Projected number of natural deaths                                       |
+| natdeaths_hivpop | p_natural_deaths_hiv | Projected number of natural deaths within HIV population                 |
+| hivdeaths        | p_hiv_deaths         | Projected HIV-related deaths by sex and age                              |
+| aidsdeaths_noart | h_aids_deaths_no_art | Projected HIV-related deaths off ART by sex, age and CD4                 |
+| aidsdeaths_art   | h_aids_deaths_art    | Projected HIV-related deaths on ART by sex, age, CD4 and treatment stage |
```

Comment:
per discussion last week — standardize these around "hiv" or "aids". (Can't remember which we preferred?)